packages/core/src/components/notification/notification.css

```diff
@@ -1,8 +1,8 @@
-@import "../../icons/icon.css";
-@import "../../icons/ui/icon-alert-circle-fill.css";
-@import "../../icons/ui/icon-check-circle-fill.css";
-@import "../../icons/ui/icon-error-fill.css";
-@import "../../icons/ui/icon-info-circle-fill.css";
+@import url("../../icons/icon.css");
+@import url("../../icons/ui/icon-alert-circle-fill.css");
+@import url("../../icons/ui/icon-check-circle-fill.css");
+@import url("../../icons/ui/icon-error-fill.css");
+@import url("../../icons/ui/icon-info-circle-fill.css");
```

Comment:
I'm not sure if the file imports need to be wrapped in url - functions 🤔

Reply:
Might not yeah but StyleLint rule `import-notation` wants it that way so...